README.md

```diff
@@ -144,3 +144,42 @@ Those charts configurations are kept in the repository.
 
 At the moment, the screenshots are made from charts using data from int.lindas.admin.ch as for some functionalities, we do not
 yet have production data.
+
+## Authentication
+
+Authentication by eIAM through a Keycloak instance.
+We use Next-auth to integrate our application with Keycloak.
+See https://next-auth.js.org/providers/keycloak for documentation.
+
+### Locally
+
+The easiest way is to run Keycloak via Docker.
+
+```
+docker run -p 8080:8080 -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin quay.io/keycloak/keycloak:19.0.3 start-dev
+```
+
+⚠️ After creating the container via the above command, if you stop it, restart the container via the docker UI so that you re-use the
+same storage, otherwise you'll have to reconfigure Keycloak.
+
+To configure Keycloak:
+
+- Access the [Keycloak admin][keycloak-admin]
```

Comment:
I just tried "admin" & "admin" and it worked 😅 But maybe it could be worth to add this info here?

Reply:
```suggestion
- Access the [Keycloak admin][keycloak-admin] (login, password: "admin", "admin")
```